example/.flowconfig

```diff
@@ -71,5 +71,3 @@ suppress_comment=\\(.\\|\n\\)*\\$FlowExpectedError
 [untyped]
 .*/node_modules/.*
 
-[version]
-^0.98.1
```

Comment:
I think this might be a matter of taste. From my commit message:

```
    fix: remove version from flowconfig
    
    we are flow checking the example, and it either works or not, it is
    silly to have a flow error because you are using a more recent version
```

It's a war between npm/yarn outdated and the flowconfig version. It works fine with current flow-bin but because the react-native folks can't handle semver 1 and do the silly 0.n thing you can't update flow-bin without moving this, and when I was done I had a clean outdated report :man_shrugging: